packages/itwinui-css/src/global.scss

```diff
@@ -61,3 +61,79 @@
 }
 
 @include mixins.animations;
+
+:where(.iui-field) {
+  --_iui-field-color-text: var(--iui-color-text);
+  --_iui-field-color-text-hover: var(--iui-color-text-hover);
+  --_iui-field-color-text-disabled: var(--iui-color-text-disabled);
+  --_iui-field-color-background: var(--iui-color-background);
+  --_iui-field-color-background-hover: var(--iui-color-background-hover);
+  --_iui-field-color-background-disabled: var(--iui-color-background-disabled);
+  --_iui-field-color-border: var(--iui-color-border-foreground);
+  --_iui-field-color-border-hover: var(--iui-color-border-foreground-hover);
+  --_iui-field-color-border-disabled: var(--iui-color-border-disabled);
+  --_iui-field-color-focus-outline: var(--iui-color-border-accent);
+  --_iui-field-height: var(--iui-component-height);
+  --_iui-field-focus-outline-offset: -2px;
+  --_iui-field-focus-outline-width: 2px;
```

Comment:
Has the outline increased from 1px to 2px?

Reply:
The default button has a `outline-width: 2px` currently.
![Screenshot 2024-03-28 at 10 45 27 AM](https://github.com/iTwin/iTwinUI/assets/849817/d806eb48-58b1-424c-972f-c2c48618adfd)

| Variant | Outline width | Outline offset |
| Default | `2px` | `-2px` |
| Borderless | `1px` | `-1px` |
| High visibility | `1px` | `-3px` |